### PR TITLE
Dim inverse Dropdown hover colour

### PR DESCRIPTION
**Pull request by daiyiwhale**
```
<Dropdown
        buttonClassName="button button-inverse dropdown-toggle"
        dropdownMenuClassName="dropdown-menu inverse"
        dropdownMenuListClassName="dropdown-menu-list"
        dropdownMenuListItemClassName="clickable"
        wrapperClassName="dropdown" 
        ... />
```

Previously:

Now:

:+1:

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @kennyt

**Comment by jfurrow** [comment]
The foreground color needs some adjustments as well. Check out the styles here: http://ashenden.github.io/canvas/

File: src/Dropdown/dropdown.less [hunk 1]
```diff
@@ -148,6 +148,11 @@
 
       &:hover {
         background: #f5f5f6;
+
+        .inverse {
+          background: #51596a;
+        }
```
Comment (jfurrow) [inline src/Dropdown/dropdown.less]: Let's change how this is defined. We don't want to have to set the `.inverse` class on all elements within the dropdown, just the `.dropdown` element itself.

So let's write something like:
```CSS
.dropdown {
  &.inverse {
    li {
      &.is-selectable {
        ...
      }
    }
  }
}
```

Comment (daiyiwhale) [inline src/Dropdown/dropdown.less]: done!

**Comment by daiyiwhale** [comment]
Now with white text \o/

File: src/Dropdown/dropdown.less [hunk 2]
```diff
@@ -64,10 +64,22 @@
 
     .dropdown-menu {
       z-index: 3;
-    }
 
-  }
+      &.inverse {
+
+        li {
+          &.is-selectable {
```
Comment (jfurrow) [inline src/Dropdown/dropdown.less]: #nitpick can we get a linebreak here?

File: src/Dropdown/dropdown.less [hunk 3]
```diff
@@ -64,10 +64,22 @@
...
+            color: #ffffff;
+
+            &:hover {
+              background: #51596a;
+            }
```
Comment (jfurrow) [inline src/Dropdown/dropdown.less]: #nitpick can we remove the linebreak here?

Comment (daiyiwhale) [inline src/Dropdown/dropdown.less]: done!

File: src/Dropdown/dropdown.less (hunk 2 above)
Comment (daiyiwhale) [inline src/Dropdown/dropdown.less]: done!

**Pull request merged**